### PR TITLE
Mo updates

### PR DESCRIPTION
**Pull request by usrdev**
greetings friends! 

pls2review for merge to master.

### CODE REVIEW

**Comment by usrdev** [comment]
thanks, @jmarhee

File: products/custom-ipxe.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+Packet supports passing custom iPXE scripts during provisioning, which allows you to install a custom operating system manually, or via automated kickstart.
+
+Since we don’t test any manually installed operating system, it goes without saying that you’ll need to be familiar with our [S.O.S. and rescue mode](https://support.packet.com/kb/articles/rescue-mode) services in case you need to troubleshoot your server.
+
+
+### **Step-by-Step Usage**
+
+Select the "Custom iPXE" operating system from the portal, or the custom\_ipxe slug if using the API.
+
+![](https://deskpro-cloud.s3.amazonaws.com/files/26944/43/42167SPMSRHSSMRGSQTS0-1539831490489.png)
```
Comment (zalkar-z) [inline products/custom-ipxe.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/advanced/bgp_overview.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+### Getting Started
+
+  
+To get started, please create a Project in the Packet portal or navigate to an existing project that you'd like to enable for BGP support. Click on the IPs & Networks tab and navigate to the BGP request form (as shown below).
+
+On the form, select either Global BGP or Local BGP. For Local BGP we default to using a private ASN of 65000. If you choose Global BGP, please enter your own public ASN. You may then enter an MD5 password for your BGP session. This is optional but highly recommended.
+
+Lastly, please enter a few details about your use case and deployment so our engineers can quickly evaluate and approve your request.
+
+![](https://deskpro-cloud.s3.amazonaws.com/files/26944/47/46225XADJQXCDKDDCDAR0-1539907873207.png)
```
Comment (zalkar-z) [inline products/network/advanced/bgp_overview.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/advanced/deploy_nopublicip.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+This is currently supported on the servers booting with the OS listed below: 
+
+CentOS7, Debian 8/9, openSUSE 42, Scientific Linux, RHEL 7, Ubuntu 14,16,17,18
+
+Should you attempt to use a non-supported operating system without a public IPv4 address, it will receive an error like “Public IPv4 is required ”.
+
+ In order to deploy without public IPv4 or public IPv6, uncheck those boxes and select the desired Private IPv4 subnet range. 
+
+![](https://deskpro-cloud.s3.amazonaws.com/files/26944/1877/1876446HZXMKKJQTHJACNB0-1556576596280.png)
```
Comment (zalkar-z) [inline products/network/advanced/deploy_nopublicip.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/advanced/native_vlan.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+<p>The Native VLAN feature is supported on servers with the 2-port NIC. &nbsp;Upon provisioning, all servers come up in Layer 3 mode by default. The 2-port NIC server boots up with a single bond, namely bond0 with both interfaces eth0 and eth1 configured as members of the bond. The support for this feature on 4-port NIC server such as n2.xlarge.x86 is planned.&nbsp;</p>
+
+<p>Assigning VLANs onto the server, the server network mode needs to be changed to either Layer-2 only on Mixed Layer 2/Layer 3. In the Layer-2 only mode, VLANs can be assigned to either of the interfaces, with one of the VLANs marked as &ldquo;native&rdquo;. But, in the mixed Layer 2/Layer 3 mode, VLANs can be assigned to only the eth1 interface which is outside of the bond0 interface.&nbsp;</p>
+
+<p>In order to set a VLAN as &ldquo;native&rdquo;, click on the &ldquo;Manage&rdquo; button next to it and follow the instructions.&nbsp;</p>
+
+<p><img src="https://deskpro-cloud.s3.amazonaws.com/files/26944/1877/1876413QJWKNHTAKTJJHQM0-1556575862642.png" class="fr-fic fr-dii" width="624" height="323"></p>
```
Comment (zalkar-z) [inline products/network/advanced/native_vlan.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/basic/backend_transfer.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Backend transfer within a facility is always free, since no bandwidth leaves the datacenter. When transferring data our global facilities, we have to move that traffic across our network.  As such, bandwidth is billed on a usage basis at a reduced rate of **$0.03/GB**. There are no monthly or other fees associated with Global Backend Transfer - just the cost of sending data between facilities.
+
+Please [contact us](mailto:support@packet.com) if you are expecting to be transferring large volumes of data between sites, to ensure that we can offer you the most cost-effective price on related traffic.
+
+## How do I enable Backend Transfer?
+
+In the client portal, browse to the IPs & Networks page, and then the Backend Transfer tab, for the project you wish to enable backend transfer on, and then click the toggle icon in the upper right-hand corner.
+
+![46221MMBKSYJMBGSQGYP0-1539907677144.png](https://deskpro-cloud.s3.amazonaws.com/files/26944/47/46221MMBKSYJMBGSQGYP0-1539907677144.png)
```
Comment (zalkar-z) [inline products/network/basic/backend_transfer.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/basic/layer2_overview.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+
+**ℹ️Please Note:** For the purposes of this documentation, we are using eth0/eth1 to represent the first and second NIC. The actual interface name will depend on what operating system and hardware config you are using.**
+
+
+### Bonding on Packet
+By default, each server has two interfaces that are setup in an LACP bond that is configured both in the Host OS and on the switch:  
+  
+  
+
+![](https://deskpro-cloud.s3.amazonaws.com/files/26944/47/46214WDTGXRWAGHKNHHP0-1539906413184.png)
```
Comment (zalkar-z) [inline products/network/basic/layer2_overview.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/basic/layer2_x1.small.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+### #1 - Enable Layer 2 Feature for your Project
+
+  
+
+In order to use Layer 2 on your x1.small, or any other server, you’ll need to enable Layer 2 networking for your project via our portal.  You can find this in the "IPs and Networks" tab.  
+  
+Once enabled, under "Virtual Networks" you can add one or more networks like this:
+
+![](https://deskpro-cloud.s3.amazonaws.com/files/26944/47/46218APQWHMWZMCBWHSN0-1539907054068.png)
```
Comment (zalkar-z) [inline products/network/basic/layer2_x1.small.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

File: products/network/doorman.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+You can use your favorite 2FA app, as long as is supports Time-based One-time Password Algorithm \(TOTP\) which is an open standard. Example: Google Authenticator, Authy, Duo Security, should all work fine.
+
+### Download OpenVPN Configuration Files
+
+Once you have 2FA enabled, a new option will be visible, Packet Customer VPN.
+
+After you turn it on, you will see the option to download the OpenVPN configuration files for each of Packet’s facilities.
+
+![](https://deskpro-cloud.s3.amazonaws.com/files/26944/43/42266XXHJRRCSJXBQKPR0-1539835990437.png)
```
Comment (zalkar-z) [inline products/network/doorman.md]: all images linking to deskpro in this file should be moved to /docs/images and links updated accordingly.

**Pull request merged**